### PR TITLE
[Aftershock] Alien Terrain

### PR DESCRIPTION
**Pull request by Mom-Bun**
#### Summary

SUMMARY: Mods "Aftershock: Alien Plants"

#### Purpose of change

Adding some ultra basic framework terrain and objects for the planned aftershock region mod. I'm planning on expanding further on these with more stuff as the idea of the region excites me a lot but for now I'm sticking with trying to dip my toes in the water and not go too deep yet.

#### Describe the solution

Adds a few terrain tiles like glacial ice, and lichen (Thanks to candlebury for those two!) furniture like a geothermal vent, and 3 new items and a recipe. Most of these are there as decoration for now and to add to the atmosphere of the game, but the lichen shrub provides a food item which requires processing to be edible.

#### Testing

Booted up the game, no errors, and none of the terrain and furniture acts weirdly, and the recipe and items work as intended.

### CODE REVIEW

File: data/mods/Aftershock/items/books.json [hunk 1]
```diff
@@ -218,5 +218,24 @@
...
     "time": "50 m"
+  },
+  {
+    "id": "recipe_lichenlog",
+    "type": "BOOK",
+    "name": { "str": "Applications of Salus IV Lichen", "str_pl": "copies of Applications of Salus IV Lichen" },
+    "description": "A sheaf of notes detailing some species of lichen. Most of the information is useless to you except for the fact that the majority of Salus IV lifeform's metabolism rely on a macronutrient that acts like a neurotoxin for humans. One useful note is a detailed procedure to neutralize said neurotoxin.",
+    "weight": "50 g",
+    "volume": "500 ml",
+    "price": 5000,
```
Comment (Xaleth) [inline data/mods/Aftershock/items/books.json:229]: 5000 cents or USD?

Comment (Positronic-Girl) [reply data/mods/Aftershock/items/books.json:229]: Oh, its cents, let me fix it

**Review by John-Candlebury — APPROVED** [review]
You'll have to run the json formatter, but I think these look great. Thanks for helping out!

I think in your case you can just do `make style-json` from the command line to make it run.

**Comment by Positronic-Girl** [comment]
> You'll have to run the json formatter, but I think these look great. Thanks for helping out!
> 
> I think in your case you can just do `make style-json` from the command line to make it run.

Thanks! Done so!

And yeah! I'd love to contribute more to the aftershock region project in the future 100%! the idea behind it is great, and there is a lot of potential there that just makes my head buzz with the world building possibilities, so I might end up being a frequent person in mod-development to talk about it, haha...

File: data/mods/Aftershock/items/comestibles/alienfood.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "name": "clump of lichen",
+    "weight": "10 g",
+    "color": "red",
+    "use_action": [ "BLECH" ],
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 400,
+    "healthy": -20,
+    "quench": 15,
+    "description": "A clump of moist squishy lichen, you get the impression that it might not be the safest thing to eat considering how it makes your hand tingle.",
```
Comment (aoae23) [inline data/mods/Aftershock/items/comestibles/alienfood.json:14]: Maybe avoid comma splices - could be changed to `lichen.  You...`

File: data/mods/Aftershock/maps/terrain_floraxeno.json [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+      "sound_fail": "squish.",
+      "ter_set": "t_lichendirt",
+      "items": [ { "item": "yum_lichen", "count": [ 1, 2 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_lichen_harvested",
+    "name": "lichen shrub",
+    "description": "A small mound of lichen grows here, noticeably bigger than the lichen and moss around it. A few clumps have been removed",
```
Comment (aoae23) [inline data/mods/Aftershock/maps/terrain_floraxeno.json:102]: Period

**Pull request merged**